pkg/daemon/ceph/agent/flexvolume/util.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flexvolume
+
+import (
+	"os/exec"
+	"strings"
+)
+
+// GetRookFlexBinaryPath returns the path of rook flex volume driver
+func GetRookFlexBinaryPath() string {
```

Comment:
A couple minor suggestions:
- If a method is called from the same package in which it's defined, it's preferred to name it with lower case to make it private, such as `getRookFlexBinaryPath()`
- Would you mind defining this method in server.go? It goes along fine with the other methods there.

Reply:
sure

Reply:
done